Submitted upstream: https://gitlab.gnome.org/GNOME/gnome-flashback/-/merge_requests/64

From eda3cf40edb15788f0d3890d0492a4f3ee5e1291 Mon Sep 17 00:00:00 2001
From: oreo639 <oreo6391@gmail.com>
Date: Fri, 19 Apr 2024 16:15:01 -0700
Subject: [PATCH] sessions: start gnome-flashback in gnome-flashback.session

This partially reverts 43fbc58d84d9a4f8e4ba74b6029e17a0171c5e4c, and fixes
starting gnome-flashback without systemd init.

Fixes: #97
---
 data/sessions/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/sessions/Makefile.am b/data/sessions/Makefile.am
index d35b38a..547118f 100644
--- a/data/sessions/Makefile.am
+++ b/data/sessions/Makefile.am
@@ -1,6 +1,7 @@
 NULL =
 
 required_components_common = \
+	gnome-flashback \
 	gnome-panel \
 	$(NULL)
 
-- 
GitLab

